week08/guess the word.cpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * // This is the Master's API interface.
+ * // You should not implement it, or speculate about its implementation
+ * class Master {
+ *   public:
+ *     int guess(string word);
+ * };
+ */
+class Solution
+{
+    ///Time Complexity O(10)

```

Comment:
There is nothing in this world called as `O(10)`. It should be O(1). But in this case, it should be O(N) where N is the number of words in the words list.